src/bin/languageserver/mod.rs

```diff
@@ -669,7 +669,7 @@ impl SolangServer {
                 SolangServer::construct_expr(expr1, lookup_tbl, symtab, ns);
             }
 
-            ast::Expression::Ternary(_locs, _typ, expr1, expr2, expr3) => {
+            ast::Expression::Conditional(_locs, _typ, expr1, expr2, expr3) => {
```

Comment:
If don't like this name, because a `if(condition)` is also a conditional. How about `TernaryConditional`?

Reply:
It is known in many languages as the conditional operator: https://en.wikipedia.org/wiki/Ternary_conditional_operator

How do you feel about `Expression::ConditialOperator()`

Reply:
That is a good option